docs/content.zh/docs/dev/table/sql/show.md

```diff
@@ -913,10 +913,17 @@ SHOW PROCEDURES [ ( FROM | IN ) [catalog_name.]database_name ] [ [NOT] (LIKE | I
 ## SHOW VIEWS
 
 ```sql
-SHOW VIEWS
+SHOW VIEWS [ ( FROM | IN ) [catalog_name.]database_name ] [ [NOT] LIKE <sql_like_pattern> ]
 ```
 
-展示当前 catalog 和当前 database 中所有的视图。
+Show all views for an optionally specified database. If no database is specified then the views are returned from the current database. Additionally, the output of this statement may be filtered by an optional matching pattern.
+
+**LIKE**
+Show all views with given view name and optional `LIKE` clause, whose name is whether similar to the `<sql_like_pattern>`.
+
+The syntax of sql pattern in `LIKE` clause is the same as that of `MySQL` dialect.
+* `%` matches any number of characters, even zero characters, `\%` matches one `%` character.
+* `_` matches exactly one character, `\_` matches one `_` character.
```

Comment:
Do we want to provide examples?  Or is it similar enough to SHOW TABLES that it'd be redundant?

Reply:
>Or is it similar enough to SHOW TABLES that it'd be redundant?

yep, I had exactly this thought, may be we can explicitly say that `SHOW TABLES` examples could be also used as examples for `SHOW VIEWS`

Reply:
Yeah, maybe that is obvious?  Not sure how pedantic to be in docs.